### PR TITLE
data: piyush-multiplexer

### PR DESCRIPTION
**Pull request by piyush-multiplexer**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4088"></a>

### CODE REVIEW

File: data/PiyushGoyani.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Piyush Goyani",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Software Engineer(Majorly Front), Occasionally Python Coder, ML Enthusiastic, Plant Based Programmer",
+  "avatar": "https://github.com/SaraJaoude.png",
+  "links": [
```
Comment (devvsakib) [inline data/PiyushGoyani.json:7]: you forgot to add tags.

example:
  "tags": ["Open Source", "Java", "React"],

**Review by devvsakib — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please rename your file to your `username.json`, else some features may not work for you. 

It should be changed from `PiyushGoyani.json`  - >  `piyush-multiplexer.json`.

File: data/PiyushGoyani.json (hunk 1 above)
Comment (piyush-multiplexer) [reply data/PiyushGoyani.json]: Added

**Comment by piyush-multiplexer** [comment]
> Please rename your file to your `username.json`, else some features may not work for you.
> 
> It should be changed from `PiyushGoyani.json` - > `piyush-multiplexer.json`.

Renamed

File: data/piyush-multiplexer.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Piyush Goyani",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Software Engineer(Majorly Front), Occasionally Python Coder, ML Enthusiastic, Plant Based Programmer | Blogger",
+  "avatar": "https://github.com/SaraJaoude.png",
```
Comment (kumarsonsoff3) [inline data/piyush-multiplexer.json:6]: Please update it with your avatar url! 
```suggestion
  "avatar": "https://github.com/piyush-multiplexer.png",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/piyush-multiplexer.json (hunk 2 above)
Comment (piyush-multiplexer) [reply data/piyush-multiplexer.json]: Added

File: data/piyush-multiplexer.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Piyush Goyani",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Software Engineer(Majorly Front), Occasionally Python Coder, ML Enthusiastic, Plant Based Programmer | Blogger",
+  "avatar": "https://github.com/piyush-multiplexer.png",
```
Comment (Panquesito7) [inline data/piyush-multiplexer.json:6]: The avatar field is no longer required as of a new LinkFree update.
```suggestion
  "bio": "Full Stack Software Engineer(Majorly Front), Occasionally Python Coder, ML Enthusiastic, Plant Based Programmer | Blogger"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/piyush-multiplexer.json (hunk 3 above)
Comment (piyush-multiplexer) [reply data/piyush-multiplexer.json]: Removed. Thanks for letting me know. @kumarsonsoff3 please note!

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**